media/engine/webrtc_voice_engine.cc

```diff
@@ -2582,4 +2592,29 @@ void WebRtcVoiceMediaChannel::ConfigureEncoders(const webrtc::AudioEncoder::Conf
   }
 }
 
+void WebRtcVoiceMediaChannel::GetAudioLevels(
+    cricket::AudioLevel* captured_out,
+    cricket::ReceivedAudioLevel* received_out,
+    size_t received_out_size,
+    size_t* received_size_out) {
+
+  cricket::AudioLevel captured = 0;
+  for (const auto& kv : send_streams_) {
+    captured = kv.second->GetAudioLevel();
```

Comment:
The expectation is that there will only really be one send stream?

Reply:
Yes.